tests/Model/Customer/CustomerListTest.php

```diff
@@ -554,8 +554,139 @@ public function testSingleSearchQueryFilter() {
         $modifiedListing = $handler->getListing();
         $this->assertEquals(5, $modifiedListing->getCount());
 
-    }
+        // -- Tests for Exact Search (Field should must match exactly)
+
+        // ------------
+        $listing = new Customer\Listing();
+        $handler = new FilterHandler($listing);
+
+        $searchFilter = new SearchQuery(['firstname'], '"jane"');
+        $handler->addFilter ($searchFilter);
```

Comment:
```suggestion
        $handler->addFilter($searchFilter);
```
there are some spaces to be cleared